src/stores/businessBootstrap.ts

```diff
@@ -0,0 +1,130 @@
+import { StatusCodes } from 'http-status-codes'
+import { FilingTypes } from '@bcrs-shared-components/enums'
+import {
+  CorpTypeCd, GetCorpFullDescription, GetCorpNumberedDescription
+} from '@bcrs-shared-components/corp-type-module'
+import { filingTypeToName } from '~/utils/todo/task-filing/helper'
+
+/** Manages bcros bootstrap business (temp reg) data */
+export const useBcrosBusinessBootstrap = defineStore('bcros/businessBootstrap', () => {
+  const bootstrapFiling: Ref<{ filing: BootstrapFilingI }> = ref(undefined)
+  const bootstrapIdentifier = computed(() => bootstrapFiling.value?.filing.business.identifier)
+  const bootstrapLegalType = computed(() => bootstrapFiling.value?.filing.business.legalType)
+  const bootstrapFilingType = computed(() => bootstrapFiling.value?.filing.header.name)
+  const bootstrapNrNumber = computed(() =>
+    bootstrapFiling.value?.filing[bootstrapFilingType.value]?.nameRequest.nrNumber)
+  const bootstrapFilingDisplayName = computed(() => {
+    if (!bootstrapFiling.value) {
+      return undefined
+    }
+    const filingName = filingTypeToName(
+      bootstrapFilingType.value,
+      undefined,
+      bootstrapFiling.value.filing[bootstrapFilingType.value].type)
+
+    if (bootstrapFilingType.value === FilingTypes.AMALGAMATION_APPLICATION) {
+      return filingName
+    }
+    const extraDesc = bootstrapLegalType.value === CorpTypeCd.SOLE_PROP ? ' / Doing Business As (DBA) ' : ' '
```

Comment:
This should be in lang and translated (except for the /)

Reply:
updated